assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -132,10 +146,41 @@ function FontPickerContainer({ value, onSelect, onClose }) {
   useFocusOut(ref, onClose, [onClose]);
 
   // Scroll to offset for current value
-  const currentOffset = fonts.findIndex(({ name }) => name === value);
+  const [currentOffset, setCurrentOffset] = useState(
+    fonts.findIndex(({ name }) => name === value)
+  );
```

Comment:
Hm, I guess `value` will never be updated from "on higher", but this does read a little lacking. If it did update from parent,`currentOffset` would not be updated.

Reply:
This used to just be a straight variable, but this just moves it into a setter. No logic has changed here. And the reason for that is if we sort or filter the list and the offset now points to an index that isn't in the list it bombs. So this let's us `-1` the offset when we need to prevent that.